react/features/aeternity/components/TipButton.js

```diff
@@ -174,10 +174,10 @@ class TipButton extends Component<Props, State> {
      * @returns {void}
      */
     _onChangeValue({ target: { value } }) {
-        const validationRegExp = /^\d*\.?\d*/;
+        const validationRegExp = /\d+\.?\d*/;
```

Comment:
```suggestion
        const validationRegExp = /^\d+(\.\d+)?\d*$/;
```

Reply:
I'm not sure about this: `(\.\d+)`. It looks like it will crash validation

Reply:
not sure what it breaks but
```js
r.test(0)
// true
```
so I will add the check for the zero here or somewhere

Reply:
> I'm not sure about this: `(\.\d+)`. It looks like it will crash validation

why it will crash? i tested it
if you are concerned about capturing groups (tough it should work anyway) use this 
`^\d+(?>\.\d+)?\d*$`

of course minimum value should be checked additionally (like value>0)

Reply:
i proposed the change because you already modified the regex, so it is better to be fixed fully